waiter/src/waiter/auth/oidc.clj

```diff
@@ -20,9 +20,11 @@
             [clojure.string :as str]
             [clojure.tools.logging :as log]
             [digest]
+            [schema.core :as s]
```

Comment:
Schema validation should be reserved for configs.  I understand the desire to not repeat "strict" and "relaxed" but the dependency should flow the other way, if at all.  I would prefer to respecify the list of valid options and remove the schema imports.

Reply:
Removed dependency schema packages.